<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sim_U4_T4</class>
 <widget class="QWidget" name="sim_U4_T4">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sim_U4_T4</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#sim_U4_T4,
QScrollArea#scrollArea{
background-color:rgba(0,0,0,0);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1006</width>
        <height>238</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#scrollAreaWidgetContents{
background-color:rgb(0,0,0);
}
QLabel{
color:white;
}
QTableWidget::item {
background-color: black;
color: white;
border: 1px solid rgb(80, 80, 80); 
}
QTableWidget::item:selected {
background-color: rgb(0,51,112);
}
QTableWidget QWidget,
QTableWidget QFrame{
background-color: rgb(173,51,51);
}
QPushButton{
font:16pt &quot;Britannic Bold&quot;;
color:white;
background-color:rgba(0,51,102,200);
}
QPushButton:hover{
background-color:rgba(0,51,150,200);
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Elementos Estructurantes</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="CustomPlotWidget" name="cp_1" native="true">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="CustomPlotWidget" name="cp_2" native="true">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="CustomPlotWidget" name="cp_3" native="true">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Elemento Estructurante</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cb_estruc">
           <item>
            <property name="text">
             <string>Diamante</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Disco</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Línea</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Cuadrado</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Rectangulo</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lb_r">
           <property name="text">
            <string>Radio</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sb_r">
           <property name="suffix">
            <string>px</string>
           </property>
           <property name="minimum">
            <number>2</number>
           </property>
           <property name="maximum">
            <number>999</number>
           </property>
           <property name="value">
            <number>11</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="fr_grados">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Grados</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="sb_deg">
              <property name="suffix">
               <string>°</string>
              </property>
              <property name="minimum">
               <number>-360</number>
              </property>
              <property name="maximum">
               <number>360</number>
              </property>
              <property name="singleStep">
               <number>10</number>
              </property>
              <property name="value">
               <number>45</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="fr_ancho">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Ancho</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="sb_ancho">
              <property name="suffix">
               <string>px</string>
              </property>
              <property name="minimum">
               <number>2</number>
              </property>
              <property name="maximum">
               <number>999</number>
              </property>
              <property name="value">
               <number>15</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_3">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Operación</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cb_oper">
              <item>
               <property name="text">
                <string>Erosionar</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Dilatar</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Dilatar + Erosionar (Closing)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Erosionar + Dilatar (Opening)</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bt_aplicar">
           <property name="text">
            <string>Aplicar operación</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="CustomPlotWidget" name="cp_4" native="true">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="CustomPlotWidget" name="cp_5" native="true">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="CustomPlotWidget" name="cp_6" native="true">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="CustomPlotWidget" name="cp_7" native="true">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomPlotWidget</class>
   <extends>QWidget</extends>
   <header>clases.PlotWidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
